doc/run.md

```diff
@@ -273,4 +306,4 @@ production Mayastor deployment and operation instructions.
 [lxd]: https://linuxcontainers.org/
 [libvirtd]: https://libvirt.org/index.html
 [terraform-readme]: ./terraform/readme.adoc
-[aarch64-branch]: 
\ No newline at end of file
+[aarch64-branch]:
```

Comment:
Whoops I think this line is left in by mistake, could you trim it? :sweat:

Reply:
Done.